.gitignore

```diff
@@ -26,3 +26,5 @@ yarn-error.log
 !.yarn/releases
 !.yarn/sdks
 !.yarn/versions
+.yalc/
+yalc.lock
```

Comment:
I don't have a strong opinion on this (and I was the one suggesting adding it to the gitignore) but the team decided not to add yalc here when the subject came up.

Better do it in a different PR if we want to in either case